### PR TITLE
Add gamebosu and edit install method page to new design

### PR DESCRIPTION
**Pull request by HelloYeew**
- Add gamebosu
- Install method page

### CODE REVIEW

File: content/pages/install/index.md [hunk 1]
```diff
@@ -8,14 +8,16 @@ summary: "Instruction to install and update rulesets to osu!lazer."
 
 # download rulesets file
 
-Go to the GitHub release page of the desired ruleset. A table of rulesets and their download links can be found here [rulesets status page](https://rulesets.info/pages/status).
+Get rulesets file from status page by go to [rulesets status page](https://rulesets.info/pages/status) and click on `detail & download` button on desired rulesets.
```
Comment (IKeepItNoodles) [inline content/pages/install/index.md:11]:
```suggestion
Get rulesets file from status page by going to [rulesets status page](https://rulesets.info/pages/status) and then clicking on the `detail & download` button for the desired ruleset.
```

Comment (LumpBloom7) [inline content/pages/install/index.md:11]:
```suggestion
Ruleset files can be obtained at [rulesets status page](https://rulesets.info/pages/status) and clicking `detail & download` button on the desired ruleset.
```

File: content/pages/install/index.md [hunk 2]
```diff
@@ -8,14 +8,16 @@ summary: "Instruction to install and update rulesets to osu!lazer."
...
 
 ![status page](img/status-download.png)
 
-Then download the ruleset by selecting the `.dll` file in the release list and downloading that file only.
+A box with a changelog and select ruleset file detail will appear. After finish read about changelog, then click `Download` button at the right-bottom corner of the box to get a ruleset file.
```
Comment (LumpBloom7) [inline content/pages/install/index.md:15]: A dialog will appear with a changelog. Click the `Download` button at the bottom-right corner to initiate the download.

File: content/pages/install/index.md [hunk 3]
```diff
@@ -8,14 +8,16 @@ summary: "Instruction to install and update rulesets to osu!lazer."
...
 
 ![sentakki release download](img/sentakki-release-download.png)
 
+A link to download and chnagelog will auto-update every time a ruleset release a new version so you can grab a new release every time a ruleset have updated and not to worry about out-of-date link.
```
Comment (LumpBloom7) [inline content/pages/install/index.md:19]: The changelog and download link will automatically update when a ruleset is updated.

File: content/pages/install/index.md (hunk 2 above)
Comment (IKeepItNoodles) [inline content/pages/install/index.md:15]:
```suggestion
A box with the changelog of the selected ruleset file details will appear. After you finish reading about changelog click the `Download` button at the right-bottom corner of the box to get the ruleset file.
```

File: content/posts/gamebosu/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- Last Update : 2021.514.0
+- Can import and play without error : Yes
+
+# overview
+
+- Creator : [Game4all](https://github.com/Game4all)
+- Availability : Available Publicly
+- GitHub Repositories : [Game4all/gamebosu](https://github.com/Game4all/gamebosu)
+
+Gamebosu is a rulesets that can make osu! play gameboy. The ruleset is developed by Game4all.
```
Comment (LumpBloom7) [inline content/posts/gamebosu/index.md:20]: Gamebosu is a ruleset that emulates a gameboy, and allows for playing gameboy ROMs. The ruleset is developed by Game4All.

File: content/posts/gamebosu/index.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+# gameplay
+
+Player can install gameboy or gameboy color ROM files and play it on this rulesets. Play by select any song in the game, let the song continue and play gameboy! (Even the song end, you can play it until you quit a song). See install rom method [here](https://github.com/Game4all/gamebosu#installation-instructions).
```
Comment (LumpBloom7) [inline content/posts/gamebosu/index.md:24]:
```suggestion
Player can install gameboy or gameboy color ROM files and play it on this rulesets. You can start playing by selecting any beatmap in the game! (You can continue playing even after the beatmap ends). See ROM installation method [here](https://github.com/Game4all/gamebosu#installation-instructions).
```

File: content/posts/gamebosu/index.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+# mods
+
+No mods available
+
+# settings
+
+Player can adjust
+
+- `Gameplay Clocl rate` by a slider
```
Comment (LumpBloom7) [inline content/posts/gamebosu/index.md:51]:
```suggestion
- `Gameplay Clock rate` via a slider
```

File: content/posts/gamebosu/index.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+- `Lock gameboy clock rate` by a toggle button
```
Comment (LumpBloom7) [inline content/posts/gamebosu/index.md:52]:
```suggestion
- `Lock gameboy clock rate` via a toggle button
```

File: content/posts/gamebosu/index.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+- `Prefer Gameboy Color mode when launching original gameboy ROMs` by a toggle button
+- `Gameboy Scale` by a slider
```
Comment (LumpBloom7) [inline content/posts/gamebosu/index.md:54]:
```suggestion
- `Gameboy Scale` via a slider
```

File: content/posts/gamebosu/index.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+- `Enable Sound Playback (VERY EXPERIMENTAL)` by a toggle button
```
Comment (LumpBloom7) [inline content/posts/gamebosu/index.md:55]:
```suggestion
- `Enable Sound Playback (VERY EXPERIMENTAL)` via a toggle button
```

File: content/posts/gamebosu/index.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+and player can `Open ROMs folder` and `Delete ROM save data` from settings menu too.
```
Comment (LumpBloom7) [inline content/posts/gamebosu/index.md:57]:
```suggestion
The player is able to `Open ROMs folder` and `Delete ROM save data` from settings menu too.
```

**Review by LumpBloom7 — CHANGES_REQUESTED** [review]

File: content/posts/gamebosu/index.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+Gamebosu is a ruleset that emulates a gameboy, and allows for playing gameboy ROMs. The ruleset is developed by Game4All.
+
+# gameplay
+
+Player can install gameboy or gameboy color ROM files and play it on this rulesets. You can start playing by selecting any beatmap in the game! (You can continue playing even after the beatmap ends). See ROM installation method [here](https://github.com/Game4all/gamebosu#installation-instructions).
+
+# controls
+
+A control is based on gameboy and player can only play on keyboard now.
```
Comment (IKeepItNoodles) [inline content/posts/gamebosu/index.md:28]:
```suggestion
Controls are based on gameboy and player can only play using keyboard at the moment.
```

**Review by LumpBloom7 — APPROVED** [review]

**Pull request merged**